src/core_modules/capture-core/components/Pages/ViewEvent/epics/getCategoriesDataFromEvent.js

```diff
@@ -10,12 +10,12 @@ function getCategoryOptionsAsync(optionIds: string) {
             fields: 'id,displayName,categories~pluck,access',
             filter: `id:in:[${optionIds}]`,
         })
-        .then(response => response && response.categoryOptions);
+        .then(response => response?.categoryOptions);
 }
 
 
 export async function getCategoriesDataFromEventAsync(event: CaptureClientEvent): Promise<?Array<Object>> {
-    const optionIdsFromEvent = event.attributeCategoryOptions;
+    const optionIdsFromEvent = event.attributeCategoryOptions?.replace(/;/g, ',');
```

Comment:
Hey @EirikHaugstulen, 

Do you want to replace all the occurrences of`;` with`,`?
I think that `;` needs to be escaped like this `.replace(/\;/g, ",")`. You can also use `replaceAll(";", ",")` and simplify the regex syntax. 🤔

Reply:
Hey @simonadomnisoru! 
Thanks for reviewing. I also thought the `;` should be escaped, but ESLint does not agree.
Do you still want me to change the code?

<img width="935" alt="image" src="https://user-images.githubusercontent.com/12842868/125584046-e279872c-50c8-451a-a15f-1a8ad1e418bf.png">